### PR TITLE
[orchestrator] add explorter retention

### PR DESCRIPTION
**Pull request by fisherevans**

### CODE REVIEW

File: content/en/developers/guide/data-collection-resolution-retention.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ Find below a summary of Datadog data [collection][6], [resolution][5], and reten
...
 | Infrastructure                      | Events                                                                            | Datadog Agent, integrations, or API                                                                                                   | Real time                                                                                      | 1 second              | 15 months                                                                                                                      |
+| Infrastructure                      | Orchestrator Explorer (Kubrenetes)                                                | Datadog Agent                                                                                                                         | 15 seconds                                                                                     | 15 seconds            | 15 minutes                                                                                                                     |
```
Comment (drichards-87) [inline content/en/developers/guide/data-collection-resolution-retention.md:45]:
```suggestion
| Infrastructure                      | Orchestrator Explorer (Kubernetes)                                                | Datadog Agent                                                                                                                         | 15 seconds                                                                                     | 15 seconds            | 15 minutes                                                                                                                     |
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
@fisherevans Left some minor feedback for spelling.

**Comment by drichards-87** [comment]
@fisherevans I'll go ahead and commit the change I suggested (spelling typo). After I do so, can I merge the PR? Or are you waiting for additional reviews?

**Comment by fisherevans** [comment]
Go for it @drichards-87 - I've got a general 👍 from the team

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**